conda-forge.yml

```diff
@@ -1,8 +1,9 @@
+build_platform:
+  osx_arm64: osx_64
```

Comment:
You are adding an arch that wasn't there before. You had linux aarch64. See https://github.com/conda-forge/gsw-feedstock/blob/main/conda-forge.yml for an example on how to re-add that.

Reply:
oh sure, sorry, that was the wrong arch...let me try again...